Helper/Order.php

```diff
@@ -1046,11 +1046,21 @@ public function getExistingOrder($orderIncrementId)
     protected function quoteAfterChange($quote)
     {
         $quote->setUpdatedAt($this->date->gmtDate());
+        // If it's PPC quote make it temporary active
+        // for third party plugins work
+        $is_quote_active = $quote->getIsActive();
+        if (!$is_quote_active) {
+            $quote->setIsActive(true);
+        }
```

Comment:
How about just `$quote->setIsActive(true);`, without the condition?

Reply:
I don't know, I'd prefer to save the condition if you don't mind.